### DISCUSSION TITLE
Narrow Results Source to the Skills field under LocalPeopleResults

### DISCUSSION THREAD

**Discussion by mschillinghaas** [microsoft-search/pnp-modern-search#3218] [Q&A] [answered]
I'm trying to narrow the users search to just the Sharepoint user Field SPS-Skills. I want search to only query that filed and return results.

**Comment by mschillinghaas** [comment]
I can have users put in Skills="sharepoint" or whatever they are looking for under the skills field and this works, however i would rather just narrow the search to that field so they can just type in sharepoint and it will find all users that have sharepoint listed under their Skills field in Delve.

**Comment by kasperbolarsen** [comment, marked as answer]
Like this ? 
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/38ff6e49-2f82-418f-bf65-fe97df1b3482)

**Comment by mschillinghaas** [comment]
You’re a genius…thank you

[Haas_Automation]

Mike Schilling
System Engineer III
(805) 278-1800 x7333
***@***.******@***.***>
Using HelloID yet?  Try it at: https://haas.helloid.com<https://haas.helloid.com/>
Visit our “Self-Service Portal” site: https://support.haascnc.com<https://support.haascnc.com/>
